src/components/ComposerQuill/composer.js

```diff
@@ -436,6 +436,17 @@ class Composer extends React.Component {
   handleFocus() {
     this.quill.focus();
 
+    let text = this.quill.getText();
+  // temporary solution for draft bullet point gone after space switch
+
+  while (text.match(/\* /)) {
+      const indexMatch = text.match(/\* /).index;
+
+      this.quill.deleteText(indexMatch, 2);
+      this.quill.formatLine(indexMatch, 'list', 'bullet');
+      text = this.quill.getText();
+    }
```

Comment:
It's not just bullet points. All markdown was being stripped when this option was enabled. Can you verify that other things either A) don't get replaced inline or B) get replaced but still send correctly?

In an ideal world, everything would work as it does today, but when we copy/paste data _only then_ would the rich text be shown. I kind of doubt that is possible.

Reply:
These few lines are handling draft message change for bullet points when space switches. For message strip away when send rich text for that , I added rich text parsing for bold, italics, list. Yeah it doesn't support code block (inline text) right now but it is possible to expand to other parsing such as inline too. I just wanna review this approach before implement it for other rich text format.  Also, I given look to other libraries that can help in parsing rich text of quill format to markdown but didn't find anyone. So, I used this approach.

Reply:
This seems expensive to do in focus. I know focus is called a bunch especially in accessibility so I'm a little worried that it'll slow down the composer when just keying around. 

The solution overall seems complex and I think it's weird that we have to manually handle it. I would think quill already has handling for this. If you play on quill's demo, they allow you to copy and paste lists just fine. So I wonder if we implemented it wrong somehow and we need to figure out why?

Reply:
Yeah, It seem expensive to me too. I didn't find other solution and the copy paste is not a problem. It works without these changes but when we send paste content then formatted message strip away so to retain that i did these changes to convert quill rich text format to markdown. I am open to have a meeting with you so we can work around and find best solution for it. Is it okay @phatv if i arrange meeting this Friday?

Reply:
Yep, that's good with me. I'm not sure if we'll find anything but i'm happy to try

Reply:
I moved handlefocus changes in componentDidUpdate. It works there also as suggested by @phatv .